plugins/SitesManager/templates/_siteWithoutDataTabs.twig

```diff
@@ -101,6 +107,36 @@
         {% endif %}
     </div>
 
+    {% if cloudflare %}
+        <div id="cloudflare" class="col s12">
+
+            <p></p><p></p>
+            <div class="system notification notification-info">
+                {{ 'SitesManager_CloudflareDetected'|translate('<a target="_blank" rel="noreferrer noopener" href="{CLOUDFLARE_URL}">','</a>')|raw }}
+            </div>
+
+            <div class="right">
+                <img src="plugins/SitesManager/images/cloudflare-icon.png" style="height: 32px;width: 32px;">
+            </div>
+            <p>{{ 'SitesManager_SiteWithoutDataCloudflareIntro'|translate }}</p>
+            <br>
+            <p>{{ 'SitesManager_SiteWithoutDataCloudflareFollowStepsIntro'|translate}}</p>
+            <p>1. {{ 'SitesManager_SiteWithoutDataCloudflareFollowStep1'|translate('<a target="_blank" rel="noreferrer noopener" href="https://dash.cloudflare.com/">','</a>')|raw }}</p>
+            <p>2. {{ 'SitesManager_SiteWithoutDataCloudflareFollowStep2'|translate }}</p>
+            <p>3. {{ 'SitesManager_SiteWithoutDataCloudflareFollowStep3'|translate }}</p>
+            <p>4. {{ 'SitesManager_SiteWithoutDataCloudflareFollowStep4'|translate }}</p>
+            <p>5. {{ 'SitesManager_SiteWithoutDataCloudflareFollowStep5'|translate }}</p>
+            <p>6. {{ 'SitesManager_SiteWithoutDataCloudflareFollowStep6'|translate }}</p>
+            <p>7. {{ 'SitesManager_SiteWithoutDataCloudflareFollowStep7'|translate }}</p>
+            <p>&nbsp;&nbsp;&nbsp;&nbsp;Matomo URL: <code vue-directive="CoreHome.SelectOnFocus">{{ piwikUrl }}</code></p>
+            <p>&nbsp;&nbsp;&nbsp;&nbsp;{{ 'SitesManager_EmailInstructionsYourSiteId'|translate('<code vue-directive="CoreHome.SelectOnFocus">' ~ idSite ~ '</code>')|raw }}</p>
+            <p>8. {{ 'SitesManager_SiteWithoutDataCloudflareFollowStep8'|translate }}</p>
+            <p>9. {{ 'SitesManager_SiteWithoutDataCloudflareFollowStep9'|translate }}</p>
```

Comment:
Would using an ordered list (`<ol>`) instead of manual numbering format things a little more nicely here? I think it might make it so that you don't have to use `&nbsp;` to indent the extra lines under line 7 as well.

Reply:
Changed to `ol`, please check again